_includes/master/kubernetes/fetch_k8s_binaries.sh

```diff
@@ -0,0 +1,22 @@
+#!/bin/bash
+k8s_ver="v1.6.2"
+
+if [ $# -eq 0 ]; then
+    binaries="kubectl kube-apiserver kube-controller-manager kubelet kube-proxy kube-scheduler"
+else
+    binaries="$@"
+fi
+
+mkdir -p k8s
+
+for x in $binaries; do
+  ver_binary=${x}-$k8s_ver
+  if [ ! -f k8s/${ver_binary}.downloaded ]; then
+    /usr/bin/wget -O k8s/${ver_binary} https://storage.googleapis.com/kubernetes-release/release/$k8s_ver/bin/linux/amd64/$x
+    if [ $? -eq 0 ]; then
+      touch k8s/${ver_binary}.downloaded
```

Comment:
I thought about doing away with it but it is an indicator that can easily be checked to see that the download appeared to be successful.  Otherwise the failure of a download would only be seen as perhaps a partial wget in the logs.  Plus since the script can be downloaded and ran interactively the user could kill the download and then run it again later to complete the downloads.